source/administration/config-settings.rst

```diff
@@ -2320,9 +2320,18 @@ This setting defines the frequency of cluster request time logging for :doc:`../
 | This feature's ``config.json`` setting is ``"ClusterLogTimeoutMilliseconds": 2000`` with whole number input.                                                         |
 +----------------------------------------------------------------------------------------------------------------------------------------------------------------------+
 
+Enable Searching of Posts
+^^^^^^^^^^^^^^^^^^^^^^^^^
+
+This setting enables the ability for users to search messages. If not enabled, when users attempt to use the search box they get an error message.
```

Comment:
Thought about this paragraph for quite a while and, to me, it just doesn't read smoothly...

How about:
`If this setting is enabled, users can search messages.  If it's disabled, users get an error message when they attempt to use the search box.`